From: Ryan VanderMeulen <rvandermeulen@mozilla.com>
Date: Sun, 23 Aug 2026 13:55:26 -0400
Subject: Bug 1978462 - disable absl nullability specifiers.

libwebrtc is not annotated consistently enough for these to be useful.
---
 abseil-cpp/absl/base/nullability.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/abseil-cpp/absl/base/nullability.h b/abseil-cpp/absl/base/nullability.h
index ad80178adc1..b209f450cef 100644
--- a/abseil-cpp/absl/base/nullability.h
+++ b/abseil-cpp/absl/base/nullability.h
@@ -224,7 +224,8 @@
 #define ABSL_POINTERS_DEFAULT_NONNULL
 
 #if defined(__clang__) && !defined(__OBJC__) && \
-    ABSL_HAVE_FEATURE(nullability_on_classes)
+    ABSL_HAVE_FEATURE(nullability_on_classes) && \
+    0 // mozilla - stop warnings in libwebrtc for "missing a nullability type specifier"
 // absl_nonnull (default with `ABSL_POINTERS_DEFAULT_NONNULL`)
 //
 // The indicated pointer is never null. It is the responsibility of the provider
